core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -225,6 +235,51 @@ public UpdateCenter() {
     UpdateCenter(@Nonnull UpdateCenterConfiguration configuration) {
         configure(configuration);
     }
+    
+    /**
+     * Creates an update center.
+     * @param config Requested configuration. May be {@code null} if defaults should be used
+     * @return Created Update center. {@link UpdateCenter} by default, but may be overridden
+     * @since TODO
+     */
+    @Nonnull
+    public static UpdateCenter createUpdateCenter(@CheckForNull UpdateCenterConfiguration config) {
+        String requiredClassName = System.getProperty(UpdateCenter.class.getName()+".className", null);
+        if (requiredClassName == null) {
```

Comment:
It would be preferrable if it also was possible to use a context variable to specify the class to use the same behaviour/pattern as in #2323